backend/tests/test_durable_dry_run_worker_documentation.py

```
@@ -0,0 +1,136 @@
+        "before any target read",
+        "encrypted DSN ciphertext and nonce",
+        "base_schema_snapshot_uuid",
+        "schema_filter",
+        "exact succeeded base snapshot",
+        "execute_isolated_dry_run",
+        "execute_bound_live_preflight",
+        "complete_isolated_dry_run",
+        "complete_live_preflight",
+        "exact expected run state version",
+        "one fresh database statement",
+        "PostgreSQL 14–18 matrix stores",
```

Ruff (0.16.1)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)